Assets/Scripts/Audio/AudioDistanceController.cs

```diff
@@ -40,23 +39,25 @@ public class AudioDistanceController : MonoBehaviour {
 
     //Add the possible player to the list of dolls
     void OnTriggerEnter2D(Collider2D other) {
-        if (other.gameObject.CompareTag("Player") || other.gameObject.CompareTag("Doll")) {
-            Debug.Log("Added doll");
+        if (other.gameObject.CompareTag("Player")) {
```

Comment:
Again Jeanie this will be a problem.
If a non-player doll enters the collider then it won't be added to the list. 
Consider this example. 
The player switches out of a doll on a conveyor belt. The conveyor belt pushes the doll onto another conveyor belt (like in 3a). The player switches back into the doll.
As the doll has already entered the collider the OnTriggerEnter will not retrigger and the doll you just entered won't be in the doll list. Thus the player won't hear the conveyor belt.